assets/src/components/TableAssignment.js

```diff
@@ -0,0 +1,112 @@
+// modified from https://demos.creative-tim.com/material-dashboard-react/?_ga=2.12819711.913135977.1549993496-494583875.1549993496#/table
+
+import React from 'react'
+import withStyles from '@material-ui/core/styles/withStyles'
+import Table from '@material-ui/core/Table'
+import TableHead from '@material-ui/core/TableHead'
+import TableRow from '@material-ui/core/TableRow'
+import TableBody from '@material-ui/core/TableBody'
+import TableCell from '@material-ui/core/TableCell'
+import HorizontalBar from '../components/HorizontalBar'
+
+const tableStyle = theme => ({
+  table: {
+    marginBottom: '0',
+    width: '100%',
+    maxWidth: '100%',
+    backgroundColor: 'transparent',
+    borderSpacing: '0',
+    borderCollapse: 'collapse'
+  },
+  tableHeadCell: {
+    color: 'inherit',
+    fontSize: '1em'
+  },
+  tableCell: {
+    verticalAlign: 'top',
+    paddingBottom: '0px'
+  },
+  tableResponsive: {
+    width: '100%',
+    marginTop: theme.spacing.unit * 3,
+    overflowX: 'auto'
+  }
+
+})
+
+function CustomAssignmentTable(props) {
+  const {classes, tableHead, tableData} = props
+
+
+  return (
+    <div className={classes.tableResponsive}>
+      <Table>
+        {tableHead !== undefined ? (
+          <TableHead>
+            <TableRow>
+              {tableHead.map((prop, key) => {
+                return (<TableCell className={classes.tableCell + ' ' + classes.tableHeadCell}
+                                   key={key}>{prop}</TableCell>)
+              })}
+            </TableRow>
+          </TableHead>
+        ) : null}</Table>
+      <Table><TableBody>
+        {
+          tableData.map((week, key) => {
```

Comment:
Here is the logic that goes with nested tables. It still have alignment issues to be fixed